js/data.js

```diff
@@ -17,7 +17,52 @@ if (previousFormEntries === null) {
   localStorage.setItem('entry-form', JSON.stringify(data));
 }
 
+var previousView = localStorage.getItem('active-view');
+
+if (previousView === null) {
+  localStorage.setItem('active-view', 'entries-div');
+}
```

Comment:
This will work, but it means adding an additional value to localStorage you need to keep track of. 

The trick here is that you need to keep track of the current view in your data model every time the user moves to a new view, so that when the page loads and you retrieve the old data model, you know that last view that the user was on and can show that view. Thankfully, we gave you a property in your data model you can use for this specific purpose: `data.view`

Every time a user moves to a new view, you should update that property with the name of that view stored in its `data-view` attribute. Then when the page loads, you can just show that view stored in `data.view` using the `switchView` function I mentioned in my other comment.